project-bom/pom.xml

```diff
@@ -31,7 +31,13 @@
             <dependency>
                 <groupId>commons-lang</groupId>
                 <artifactId>commons-lang</artifactId>
```

Comment:
unfortunately that is not the case. we have dependencies which require both (one or the other) and they are not a interchangeable without code (that we don't own) updates  --- the `imports` are different.